### DISCUSSION TITLE
waiting for etcd restore probes in RKE2 managed cluster

### DISCUSSION THREAD

**Discussion by DipindasThayyil** [rancher/rke2#7153] [Q&A]
I'm getting the " waiting for etcd restore probes " messages after a cluster config, kubernetes version and etcd  restore  and from local box level, I'm able to list the nodes and removed the unhealthy ones

 root@prod-m3:~# /var/lib/rancher/rke2/bin/kubectl get node
NAME              STATUS   ROLES                       AGE   VERSION
prod-m3   Ready    control-plane,etcd,master   95d   v1.28.10+rke2r1
prd-w2   Ready    worker                      95d   v1.28.10+rke2r1

Under the Related Resource, Mgmt CLuster state is Updating, and rke2-server service is running in the control plan node.   Any suggestions to fix the issue? 

![image](https://github.com/user-attachments/assets/b79c6e75-1b6c-45cd-86b5-d55a30fbf151)

**Comment by brandond** [comment]
Again this is a rancher issue. And you're already discussing it on a different thread. Please do not continue opening new discussions for the same issue. Stick to the existing discussion, and do as requested - provide logs from the nodes. This is not the Rancher project and providing nothing but screenshots of the Rancher UI is not sufficient.
